AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumEventDataManager.java

```diff
@@ -158,33 +163,41 @@ static void deleteEvents(int count) {
    *
    * @return Number of rows in the event table.
    */
-  static long getEventsCount() {
+  long getEventsCount() {
     long count = 0;
     if (database == null) {
       return count;
     }
     try {
       count = DatabaseUtils.queryNumEntries(database, EVENT_TABLE_NAME);
-      willSendErrorLog = false;
+      sendErrorLogs = false;
     } catch (Throwable t) {
       handleSQLiteError("Unable to get a number of rows in the table.", t);
     }
     return count;
   }
 
+  /**
+   * Whether we are going to send error log or not.
+   */
+  boolean willSendErrorLogs() {
```

Comment:
Why is this needed since its default value? Where do we change the value .

Reply:
We use the flag in case we fail to save data into SQL so we could send error logs from database